src/glossary.rst

```diff
@@ -1767,3 +1767,17 @@ Glossary
       :term:`incomplete outputs <output completion>`, then the workflow is
       not complete and the scheduler will :term:`stall` pending manual intervention.
 
+   dummy task
+      A task which runs a trivially simple script such as ``sleep 1``,
+      ``exit 0`` or ``true``, or which uses alternative :ref:`task-run-modes.skip`
+      or :ref:`workflow-run-modes` to run such a script or avoid running a
+      script at all.
+
+      .. important::
+
+         All tasks that run in dummy or simulation mode are dummy
+         tasks, but the methods outlined above will create dummy tasks in a
+         live workflow.
```

Comment:
I'm on the fence about this. 

It makes graph focused dummy task totally separate from dummy _mode_. However the post change doesn't rule out dummy mode tasks being dummy tasks (since dummy mode essentially sets `script = true` for all tasks). So I think it's better to keep describing tasks in dummy and simulation mode as a subset.

Reply:
I'm committing this suggestion as I think this is the one you mentioned in person? (IMO the concept of dummy tasks only makes sense in live mode; there is no need to make a distinction about if tasks are dummy tasks or not in dummy/simulation mode - leave that to the philosophers)